neuralmonkey/config/parsing.py

```diff
@@ -143,84 +142,49 @@ def parse_value(string):
 
     return string
 
+def parse_config(config_file, filename=""):
```

Comment:
Tohle bych pojenoval nějak jinak. `parse_config` a `parse_file` zní moc podobně a moc to neříká o tom, co se tam děje + by to chtělo, aby to bylo v doc stringu, teď ho to má oboje stejný,